docs/project-setup.md

```diff
@@ -0,0 +1,45 @@
+# Get started
+
+## Development mode
+To launch this project in development mode you'll need to run:  
+`yarn start`
```

Comment:
Is it only `yarn start`, not `npm start` as well? Same for the other scripts.

Reply:
In fact, probably we shouldn't be using `yarn` at all, since we now have `package-lock.json` and we are not using yarn workspaces.

Will remove references to `yarn`. Thanks!